webpack.config.js

```diff
@@ -43,15 +44,16 @@ module.exports = {
   },
   externals: {
     d3: 'd3',
-    hammerjs: 'Hammer'
+    hammerjs: 'Hammer',
+    vtkjs: 'vtk.js'
```

Comment:
@aashish24  This is how vtk.js is treated as an optional dependency in the same way that d3 is optional for geojs. There are other ways to do it that can be done at build time rather than runtime if we would prefer that route.